Microsoft.Toolkit.Uwp/Helpers/HttpHelperResponse.cs

```diff
@@ -37,11 +39,41 @@ public HttpRequestResult()
         /// <summary>
         /// Gets or sets holds request Result.
         /// </summary>
-        public string Result { get; set; }
+        public IHttpContent Result { get; set; }
+
+        /// <summary>
+        /// Reads the Content as string and returns it to the caller.
+        /// </summary>
+        /// <returns>string content</returns>
+        public Task<string> GetTextResultAsync()
+        {
+            if (this.Result == null)
+            {
+                return null;
```

Comment:
Will this not throw an exception as you cannot await `null`? Perhaps `return Task.FromResult<String>(null);`